docs/blog/write-the-docs-2017-stats.rst

```diff
@@ -0,0 +1,82 @@
+.. post:: Jan 8, 2018
+   :tags: stats, year in review
+   :author: Eric Holscher
+
+Write the Docs 2017 Stats
+=========================
+
+2017 was a wonderful year for Write the Docs, it marks our 5th year of existence,
+and the community is growing into a truly global community.
+This post contains stats that we feel mark our progress at each year's end,
+so that we can look back and see how we've changed as a community.
+
+.. note:: You can also read our post from :doc:`2016 <blog/write-the-docs-2017-year-in-review>`
+
+Conferences
+===========
+
+* 400 attendees in Portland (same as in 2016)
+* 250 attendees in Prague (same as in 2016)
+* 50 attendees in Australia (New)
+
+The :doc:`conferences </conf/index>` continue to be the yearly physical manifestation of our community.
+They are the time each year where hundreds of wonderful folks gather and have a great time,
+while learning so much from each other.
+Proving each year that work can be both fun and informative is one of my greatest pleasures.
+
+We continue to publish :doc:`videos </videos>` freely to the public. 
+Doing this is more inclusive to people who cannot attend the conferences, promotes learning, and is part of our :doc:`commitment to openness </about/about-the-org>`.
+
+Slack Network
+=============
+
+* 3,000 (up from 1,200 members in 2016)
+
+Slack has been quite active this past year. We've had an amazing amount of good conversations within the global community here!
+
+Newsletter
+==========
+
+* 3,406 subscribers (up from 2,485 in 2016)
+
+Our :doc:`newsletter </newsletter>` contains curated distillations from the slack conversations that you have every day.
+
+Meetups
+=======
+
+* 3108 Members
+* 30 Meetups (up from 18 Meetups in 2016)
+* 4 Continents (same as in 2016)
```

Comment:
These numbers aren't up to date, not sure the best way to get them. I think last year I just manually counted.

Reply:
Made a spreadsheet!